pkgs/games/heroic/fhsenv.nix

```diff
@@ -29,7 +29,10 @@ buildFHSEnv {
     perl
     psmisc
     python3
+    unzip
     which
+    wine
```

Comment:
I don't agree with adding Wine in the FHS env:
- That Wine specifically is stable Wine, which isn't really recommended for games (for instance, all Unity games suffer from this bug: https://bugs.winehq.org/show_bug.cgi?id=48121)
- If someone already has a version of Wine installed system-wide, they'll have to forcefully install another version that they may not want to use
- Heroic allows you to install whatever Wine version you want, what's the point in including one here?

Also, it was removed from Lutris #224545 and Bottles #202265.